base_multi_image/models/__init__.py

```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
+# © 2009 Sharoon Thomas Open Labs Business Solutions
+# © 2015 Antiun Ingeniería S.L. - Jairo Llopis
```

Comment:
I notice that the authors here differ from the `__openerp__.py`, both in the headers and in the authors key...